frontend/src/Pages/Home.js

```diff
@@ -0,0 +1,211 @@
+import "../Styles/Home.scss";
+
+const Home = () => {
+  return (
+    <section className="home-page">
+      <div className="cta container">
+        <div className="cta-text">
+          <h1>
+            All You Need for Your <span>Wardrobe</span>
+          </h1>
+          <p>
+            Shop now to get <img src="./images/max.png" alt="" /> discount.
```

Comment:
suggestion - write something about the contents of image in alt

Reply:
Will do that. Thanks!